packages/polka/index.js

```diff
@@ -3,7 +3,8 @@ import Router from 'trouter';
 import { parse } from '@polka/url';
 
 function onError(err, req, res) {
-	let code = (res.statusCode = err.status || 500);
+	const code = typeof err.status === 'number' ? err.status : 500;
+	res.statusCode = code;
```

Comment:
Avoid code style changes in a PR

```suggestion
	let code = (res.statusCode = +err.status || 500);
```

Reply:
Hmmm... This way, `err.status === true` will crash the server, for instance.

Maybe we could actually check `err.status in http.STATUS_CODES` ?

Reply:
I reverted it in the linked commit, sorry. The only rule (from Node pov) is that it has to be >= 100. You can send down a 700 status code if you want, which means that looking up `http.STATUS_CODES` isn't really an option unfortunately